webpack.config.js

```diff
@@ -47,8 +48,11 @@ module.exports = {
       chunkFilename: "styles.css",
     }),
     new HtmlPlugin({
-      template: "./public/index.html",
+      template: "./src/index.html",
```

Comment:
I think we still want "./public/index.html" here

Reply:
That's not possible. HtmlPlugin will put the `index.html` in the `dist` folder, and then the CopyPlugin will copy `public/index.html` into the `dist` folder as well, giving an error that it already exists.

Reply:
Sorry I missed at first that this PR moves index.html from public into src. Thanks for clarifying that point too.